openssh/client/service.sls

```diff
@@ -27,6 +27,8 @@ openssh_client_config:
 
 {%- for user_name, user in client.user.iteritems() %}
 
+{%- if user_name != 'test' %}
```

Comment:
I would better introduce enabled parameter and set it to `false` in testing pillar.